metricbeat/docs/fields.asciidoc

```diff
@@ -11535,15 +11539,43 @@ The display name of the service.
 
 type: keyword
 
-The start type of the service. The possible values are `ServiceAutoStart`, `ServiceBootStart`, `ServiceDemandStart`, `ServiceDisabled`, and `ServiceSystemStart`.
+The startup type of the service. The possible values are `Automatic`, `Boot`, `Disabled`, `Manual`, and `System`.
```

Comment:
This is kind of a breaking change. Should we mention that under Breaking changes?